docs/guides/development/cli.md

```diff
@@ -231,6 +231,26 @@ Although a lot of Pack management can be done through the CLI, there are still s
 - Editing your Pack's listing page (name, icon, etc).
 
 
+## Migrating from the Web IDE
+
+It's possible to start development of a Pack in the Pack Studio Web IDE and later migrate to using the CLI. For example, to take advantage of an NPM library, which isn't possible in the Web IDE.
+
+To migrate, follow all of the steps described in the [Getting started tutorial][quickstart_cli] up until the point of running `npx coda create`. Instead, manually create a file in the same directory as the pack definition called `.coda-pack.json` with the following content:
```

Comment:
Feel free to file a task to handle this in your preferred way--a long time ago we talked about creating a CLI command like `coda link path/to/manifest.ts https://coda.io/p/12345` where you just paste the pack url and we create the json file to link your code with an existing pack.

Reply:
Will do!